src/os/user/iterate_example_test.go

```diff
@@ -0,0 +1,54 @@
+package user
+
+import (
+	"errors"
+	"fmt"
+)
+
+func ExampleIterateUsers() {
+	// Get first 20 users
```

Comment:
you may need a docstring for this to be correctly rendered in the generated docs.